packages/react-components/src/components/Search/Search.spec.tsx

```diff
@@ -0,0 +1,143 @@
+import * as React from 'react';
+import { render, fireEvent, vi } from 'test-utils';
+import noop from '../../utils/noop';
+import { ISearchProps, Search, SearchSize } from './Search';
+import styles from './Search.module.scss';
+
+const baseClass = 'search';
+const inputBaseClass = `${baseClass}__input`;
+
+const defaultProps = {
+  onChange: () => noop,
+};
+
+const renderComponent = (props: ISearchProps) => {
+  return render(<Search {...props} />);
+};
+
+describe('<Search> component', () => {
+  it('should render with default size', () => {
+    const { getByRole } = renderComponent({
+      ...defaultProps,
+    });
+
+    expect(getByRole('textbox')).toHaveClass(
+      styles[`${inputBaseClass}--compact`]
```

Comment:
I feel the inconsistency here. Why this component is rendered as a compact by default when other components in DS have a medium default size?

Reply:
☝️